apps/projects/test/projects.test.js

```diff
@@ -299,115 +336,230 @@ contract('Projects App', accounts => {
         )
       })
 
-      it('fulfill bounties and accept fulfillment', async () => {
+      it('allows users to request assignment', async () => {
+        await app.requestAssignment(repoId, issueNumber, 'QmbUSy8HCn8J4TMDRRdxCbK2uCCtkQyZtY6XYv3y7kLgDd', { from: root })
+        response = await app.getApplicant(repoId, issueNumber, 0)
+        assert.strictEqual(
+          response[0],
+          root,
+          'applicant address incorrect'
+        )
+        assert.strictEqual(
+          response[1],
+          'QmbUSy8HCn8J4TMDRRdxCbK2uCCtkQyZtY6XYv3y7kLgDd',
+          'application IPFS hash incorrect'
+        )
+      })
+
+      it('users cannot apply for a given issue more than once', async () => {
+        await app.requestAssignment(repoId, issueNumber, 'QmbUSy8HCn8J4TMDRRdxCbK2uCCtkQyZtY6XYv3y7kLgDd', { from: root })
+        assertRevert( async () =>{
+          await app.requestAssignment(repoId, issueNumber, 'QmbUSy8HCn8J4TMDRRdxCbK2uCCtkQyZtY6XYv3y7kLgDd', { from: root })
+        })
+      })
+
+      it('assign tasks to applicants', async () => {
+        await app.requestAssignment(repoId, issueNumber, 'QmbUSy8HCn8J4TMDRRdxCbK2uCCtkQyZtY6XYv3y7kLgDd', { from: root })
```

Comment:
Should we extract those IPFS hashes to a variable? since the same value is used over an over

Reply:
It's not the same hash I've modified it slightly to ensure it's getting updated in the tests.